apps/bridge/package.json

```diff
@@ -18,7 +18,7 @@
     "@pancakeswap/utils": "workspace:*",
     "@vanilla-extract/next-plugin": "^2.3.0",
     "@wagmi/core": "^0.10.11",
-    "@wormhole-foundation/wormhole-connect": "0.1.7",
+    "@wormhole-foundation/wormhole-connect": "0.3.0-beta.9-development",
```

Comment:
is there a stable version?

Reply:
waiting for them to releaae

Reply:
I'll convert the pr to draft until we deploy with the official release